app/src/main/res/layout/footer_snapshots.xml

```diff
@@ -1,17 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
will look into this. i did it mostly because I don't like the different styles where some elements have their start tag standalone and thngs like this have the scheme in the same line (but a second scheme standalone again). aesthetics. not essential or relevant to the change, ofc